test/export/test_converter.py

```diff
@@ -8,18 +8,47 @@
 
 
 class TestConverter(TestCase):
+    def _check_equal_ts_ep_converter(self, mod_func, inp):
+        mod = mod_func()
```

Comment:
pass mod instance instead of mod_func here, since some models' constructor would need argument. 
It's more general to accept a model instance.